package.json

```diff
@@ -17,7 +17,7 @@
         "cmdutil": "^1.1.0",
         "cueball": "^2.3.0",
         "extsprintf": "^1.3.0",
-        "fast": "2.8.1",
+        "fast": "git+https://github.com/joyent/node-fast.git#MANTA-5170",
```

Comment:
change this to 3.0.1 and :stamp:

Reply:
Is the v3 major version change going to cause headaches here?

```
## v3.0.0

* Bump the protocol version to 2 as a means to better deal with the CRC
  incompatibility stemming from the use of node-crc version 0.3.0. This change
  replaces the attempt at addressing this issue that was part of v2.8.1. That
  solution was found to be insufficient since there were cases where the
  calcluated CRC codes from the 0.3.0 version of node-crc were correct. With
  this release the protocol version is used to clearly determine which version
  of node-crc should be used to decode a received message and encode a
  response. Servers using this version are still compatible with fast clients
  using protocol version 1, but servers using protocol version 1 will not
  support clients using protocol version 2.
```

Reply:
> Is the v3 major version change going to cause headaches here?

Probably. I guess the two options are:

a) fork node-fast and maintain a 2.x branch
b) bump the node-moray major version

Thoughts?